release.nix

```diff
@@ -6,7 +6,7 @@ let
 
   pkgs = import <nixpkgs> {};
 
-  systems = [ "x86_64-linux" "i686-linux" "x86_64-darwin" /* "x86_64-freebsd" "i686-freebsd" */ ];
+  systems = [ "x86_64-linux" "i686-linux" "x86_64-darwin" "i686-cygwin" "x86_64-cygwin" /* "x86_64-freebsd" "i686-freebsd" */ ];
```

Comment:
np, i will extend release.nix to be able to pass extra systems to built on, for now.